Describe Prometheus limits in docs (#13699)
kyma-project/kyma: docs/03-tutorials/00-observability/obsv-01-observe-application-metrics.md

   The Prometheus UI shows a new value every 10 seconds upon refreshing the page.

4. If you don't want to proceed with the following tutorial, [clean up the configuration](obsv-05-clean-up-configuration.md).

## Limitations

> The configured memory limits of the Prometheus and Prometheus-Istio instances limit the number of time series samples that can be ingested. The amount of generated time series in a Kyma cluster is depending on the following factors:

Reviewer: ```suggestion
The configured memory limits of the Prometheus and Prometheus-Istio instances limit the number of time series samples that can be ingested. The amount of generated time series in a Kyma cluster depends on the following factors:
```

* Number of Pods in the cluster
* Number of Nodes in the cluster
* Amount of exported (custom) metrics
* Label cardinality of metrics
* Number of buckets for histogram metrics
* Frequency of Pod recreation
* Topology of the Istio service mesh

> The default resource configuration of the monitoring component in the production profile is sufficient to serve 800K time series in the Prometheus Pod and 400K time series with a retention time of 30 days in the Prometheus-Istio Pod. The evaluation profile has lower limits.

Reviewer: ```suggestion
The default resource configuration of the monitoring component in the production profile is sufficient to serve 800K time series in the Prometheus Pod, and 400K time series with a retention time of 30 days in the Prometheus-Istio Pod. The evaluation profile has lower limits.
```

Maybe also add a link to 04-operation-guides/operations/02-install-kyma#choose-resource-consumption where the profiles are presented (For more information about profiles, see...). Otherwise, users might not know what that is.

> The number of ingested time series samples can be taken from the `prometheus_tsdb_head_series` metric, which is exported by the Prometheus itself. Furthermore, the [TSDB Status](http://localhost:9090/tsdb-status) page helps to identify expensive metrics.

Reviewer: Trying to use more active voice. 
Is it the user who "takes" the number of time series? Or is that automated? If it's the user, I suggest:

```suggestion
You can see the number of ingested time series samples from the `prometheus_tsdb_head_series` metric, which is exported by the Prometheus itself. Furthermore, you can identify expensive metrics with the [TSDB Status](http://localhost:9090/tsdb-status) page.
```
